ui/app/Rif/Pages/domainsPage.js

```diff
@@ -0,0 +1,49 @@
+import { Component } from 'react';
```

Comment:
Sorry to bother with this but i think we should just put the folder names in lowercase to follow the general convention in this case, what do you think about it?

Reply:
It's ok, there i upload a fix for this.